OpenRA.Mods.Common/Traits/BotModules/BaseBuilderBotModule.cs

```diff
@@ -201,6 +201,9 @@ bool IBotRequestPauseUnitProduction.PauseUnitProduction
 
 		void IBotTick.BotTick(IBot bot)
 		{
+			if (IsTraitDisabled)
```

Comment:
`ModularBot` already checks `IsTraitEnabled()` before calling `BotTick`. Making the checks explicit here is clearer, but in this case we should make sure all the modules have it and remove the `ModularBot` check.

@reaperrr do you have any input here?

Reply:
I removed the commit changing this. Checking if the trait is enabled before calling this method is good enough imho.